Makefile

```diff
@@ -0,0 +1,21 @@
+INSTALL_PROGRAM = install
+CARGO_PROGRAM = cargo
+PREFIX = /usr
+SYSTEMD_DIR ?= $(PREFIX)/lib/systemd
```

Comment:
This could also be `pkgconf systemd --variable=systemd_system_unit_dir`, but that is fine too.